pkg/controller/installation/core_controller.go

```diff
@@ -259,6 +265,12 @@ func add(mgr manager.Manager, r *ReconcileInstallation) error {
 		if err != nil {
 			return fmt.Errorf("tigera-installation-controller failed to watch FelixConfiguration resource: %w", err)
 		}
+
+		//watch for change to primary resource LogCollector
+		err = c.Watch(&source.Kind{Type: &operator.LogCollector{}}, &handler.EnqueueRequestForObject{})
```

Comment:
This was not in the original pick #1447, was this needed?

Reply:
argh, there were merge conflicts and this was not meant to be there. Good spot, i will remove